### PR TITLE
[Pulumi] Tutorial updates

### PR DESCRIPTION
**Pull request by desteves**
### Summary

Modifies existing tutorials:

* Add multi-language support to the "Deploy a worker" tutorial
* Fixes the outline markdown headers
* Adds consistent flow of steps across the tutorials
* Adds explanation of newly introduced terms, given these are beginner tutorials
* Updates metadata
* Updates deprecated resources and resource inputs in the code snippets

*All code has been tested in my local environment*

### Documentation checklist

- [x] The [documentation style guide](https://developers.cloudflare.com/style-guide/) has been adhered to.

### CODE REVIEW

**Comment by desteves** [comment]
Unsure why https://www.cloudflare.com/sign-up reports a 403 for the link check.

**Comment by kodster28** [comment]
> Unsure why https://www.cloudflare.com/sign-up reports a 403 for the link check.

That's b/c I think it gets blocked by Turnstile so it doesn't resolve correctly.

cc: @bllchmbrs, do we have a way to update our config with ignored links?

**Comment by bllchmbrs** [comment]
@kodster28 , yup, we're aware!

3 things:
1. We're going to treat these as "success" for you all.
2. It'd be good to get a "user-agent" that we can share with you so that we get real status code.
3. We don't "fail" the check yet, because we're working through these. Should finish this week.

**Comment by kodster28** [comment]
Howdy @desteves, goal is to get this reviewed by EoW.
(keep me accountable if that's not the case :) )

**Comment by crwaters16** [comment]
Hey @desteves! Kody's unexpectedly out today, so I'm taking a look now

File: src/content/docs/pulumi/tutorial/manage-secrets.mdx [hunk 1]
```diff
@@ -107,7 +107,7 @@ Getting User settings...
 
 When you use the `esc run` command, it opens the Environment and sets the specified Environment variables into a temporary environment. After that, it uses those variables in the context of the `wrangler` command. This is especially helpful when running `wrangler` commands in a CI/CD environment but wanting to avoid storing credentials directly in your pipeline.
 
-## Add Worker secrets
+## 3, Add Worker secrets
```
Comment (crwaters16) [inline src/content/docs/pulumi/tutorial/manage-secrets.mdx]:
```suggestion
## 3. Add Worker secrets
```

File: src/content/docs/pulumi/tutorial/manage-secrets.mdx [hunk 2]
```diff
@@ -125,7 +125,7 @@ esc run -i ${ESC_ENV} -- sh -c 'echo "$TOP_SECRET" | npx wrangler secret put TOP
 
 By using an external secrets management solution, commonly used Worker secrets can be stored in a single shared Environment that is accessed by the relevant Workers. You can use shell commands with `esc` to incorporate scripting and integrate them into deployment pipelines or `make` commands. Use `esc [command] --help` for more information about the various commands available in the CLI.
 
-## Load `.dev.vars`
+## 4, Load `.dev.vars`
```
Comment (crwaters16) [inline src/content/docs/pulumi/tutorial/manage-secrets.mdx]:
```suggestion
## 4. Load `.dev.vars`
```

**Review by crwaters16 — COMMENTED** [review]
A couple small nits - need to check one more thing before hitting the ✅

**Comment by desteves** [comment]
Thank you for taking a look @crwaters16. I went ahead and accepted your suggestions.

**Comment by desteves** [comment]
@crwaters16 just checking if there's anything else needed from my end

**Review by crwaters16 — APPROVED** [review]

**Comment by crwaters16** [comment]
LGTM! Thanks for the contribution 🥳

**Pull request merged**